From: Michael Olbrich <m.olbrich@pengutronix.de>
Subject: disables docs

we don't need docs and GTK_DOC_CHECK needs gtk during autoreconf.
This patch removes this to avoid the dependency.

Not for upstream.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>

---
 Makefile.am  |    2 +-
 configure.ac |    6 ------
 2 files changed, 1 insertion(+), 7 deletions(-)

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -34,8 +34,6 @@ AC_PATH_PROG([XSLTPROC], [xsltproc])
 fi
 AM_CONDITIONAL(MAN_PAGES_ENABLED, test x$enable_man_pages = xyes)
 
-GTK_DOC_CHECK([1.3])
-
 #### gcc warning flags
 
 if test "x$GCC" = "xyes"; then
@@ -199,10 +197,6 @@ data/Makefile
 data/DeviceKit-disks.pc
 src/Makefile
 tools/Makefile
-doc/Makefile
-doc/version.xml
-doc/man/Makefile
-doc/dbus/Makefile
 policy/Makefile
 po/Makefile.in
 ])
Index: b/Makefile.am
===================================================================
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = data src doc tools policy po
+SUBDIRS = data src tools policy po
 
 # Creating ChangeLog from git log (taken from cairo/Makefile.am):
 ChangeLog: $(srcdir)/ChangeLog
